File: net/filter-content/filtering_get_subpages.cs
// Gets pages linking travel insurance as their subpage.
IDeliveryItemListingResponse<object> response = await deliveryClient.GetItemsAsync<object>(
  new ContainsFilter("elements.subpages", "travel_insurance")
);

// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
IDeliveryItemListingResponse<object> response = await deliveryClient.GetItemsAsync<object>(
  new AnyFilter("elements.subpages", "travel_insurance", "car_insurance")
);

File: php/filter-content/filtering_get_subpages.php
<?php
// Gets pages linking travel insurance as their subpage.
$items = $client->getItems((new QueryParams())
  ->contains('elements.subpages', 'travel_insurance'));

// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
$items = $client->getItems((new QueryParams())
  ->any('elements.subpages', ['travel_insurance','car_insurance']));
?>